apps/site/src/pages/account/[[...slugs]].page.tsx

```diff
@@ -75,6 +75,13 @@ const Account: AuthWallPageContent = () => {
     }
   }, [router]);
 
+  useEffect(() => {
```

Comment:
Unsure why this is necessary?

Reply:
```// otherwise sidebar component does not recognize the current page```.

That's due to `PageSidebar` component deciding which page is highlighted via `pathWithoutParams.startsWith(href)`.

So if user lands to `/account` with client-side routing, we replace it with a tab instead to make sure it's highlighted.

Probably not the ideal solution, but did not wanted to spend time refactoring `PageSidebar`. 
This is related with this comment btw https://github.com/blockprotocol/blockprotocol/pull/1257#discussion_r1169118165

Reply:
See https://github.com/blockprotocol/blockprotocol/pull/1257#discussion_r1170266924

Reply:
resolved above